test/js/color-value.js

```diff
@@ -0,0 +1,45 @@
+suite('ColorValue', function() {
+  test('Constructor should throw an error if r, g and b are not integers', function() {
+    assert.throw(function() {new ColorValue(0, "lemon", 50);}, TypeError,
+      'r, g and b must be integers.');
+    assert.throw(function() {new ColorValue(0, 50, null);}, TypeError,
+      'r, g and b must be integers.');
+    assert.throw(function() {new ColorValue(50.5, 20, 50);}, TypeError,
+      'r, g and b must be integers.');
```

Comment:
It'd be good to have a test that you can construct the ColorValue with valid arguments too.